src/App/mermaidData/databaseSwitchboard/ManagementRegimesMixin.js

```diff
@@ -39,24 +40,38 @@ const ManagementRegimesMixin = (Base) =>
         : Promise.reject(this._notAuthenticatedAndReadyError)
     }
 
+    getManagementRegimesFromApi = async function getManagementRegimesFromApi(projectId, pageNo) {
+      if (!projectId) {
+        return Promise.reject(this._operationMissingParameterError)
+      }
+
+      return this._isOnlineAuthenticatedAndReady
+        ? axios
+            .get(`${this._apiBaseUrl}/managements/`, {
+              params: {
+                exclude_projects: projectId,
+                include_fields: `project_name`,
+                page: pageNo,
+                limit: DEFAULT_QUERY_LIMIT,
+              },
+              ...(await getAuthorizationHeaders(this._getAccessToken)),
+            })
+            .then((apiResults) => apiResults.data)
+        : Promise.reject(this._notAuthenticatedAndReadyError)
+    }
+
     getManagementRegimesExcludedInCurrentProject =
-      async function getManagementRegimesExcludedInCurrentProject(projectId) {
-        if (!projectId) {
-          return Promise.reject(this._operationMissingParameterError)
+      async function getManagementRegimesExcludedInCurrentProject(projectId, pageNo = 1) {
```

Comment:
does whatever we are doing below that is new warrant a  new descriptive function name? What is this function now doing?

Reply:
that is the main function (how about a simple name `getManagementRegimes`, I split off the axios call function.

Reply:
Ah, now that you named the limit constant, it makes more sense.  Maybe `getManagementRegimesExcludedInCurrentProjectByPage` since we arent getting _all_ management regimes (other than already excluded)